src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => (a.toString() > b.toString() ? 1 : -1),
+  ) {
+    let thisLenght = this.length;
```

Comment:
The variable `thisLenght` has a typo. It should be `thisLength` to correctly represent the meaning of the variable.